### PR TITLE
add readremaining.js v0.1.0 with autoupdate config

### LINKED ISSUE #7585: [Request] Add readremaining.js
https://aerolab.github.io/readremaining.js/
https://github.com/Aerolab/readremaining.js

### PR DESCRIPTION
**Pull request by dannyxx001**
@x09326 please help me review this PR, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/Aerolab/readremaining.js
- Official website (optional, not the repository): https://aerolab.github.io/readremaining.js/
- NPM package url (optional):
- GitHub / Bitbucket popularity (required):
  - Count of stars: 471
  - Count of watchers: 26
  - Count of forks: 43
- NPM download stats (optional):
  - Downloads in the last day:
  - Downloads in the last week:
  - Downloads in the last month:
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [ ] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

close #7585, cc @patopitaluga

### CODE REVIEW

File: ajax/libs/readremaining.js/0.1.0/src/readremaining.jquery.js [hunk 1]
```diff
@@ -0,0 +1,268 @@
+/*!
+ * ReadRemaining.js 1.0.0
```
Comment (x09326) [inline ajax/libs/readremaining.js/0.1.0/src/readremaining.jquery.js]: this line said the version of this lib is 1.0.0, I think you need to check this with the author.

Comment (dannyxx001) [inline ajax/libs/readremaining.js/0.1.0/src/readremaining.jquery.js]: But the tag in [here](https://github.com/Aerolab/readremaining.js/tree/0.1.0) is 0.1.0. Which should I use?

Comment (x09326) [inline ajax/libs/readremaining.js/0.1.0/src/readremaining.jquery.js]: You can open an issue in the repo to ask the author.

Comment (dannyxx001) [inline ajax/libs/readremaining.js/0.1.0/src/readremaining.jquery.js]: The author had modified it, and I also had updated it. @x09326 Please help me review again, thank you.

File: ajax/libs/readremaining.js/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "author": "Aerolab",
+  "license": "MIT",
+  "homepage": "http://aerolab.github.io/readremaining.js/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/Aerolab/readremaining.js.git",
+    "basePath": "",
+    "files": [
+      "src/readremaining*",
+      "css/rr*"
```
Comment (x09326) [inline ajax/libs/readremaining.js/package.json]: @PeterDaveHello 
I think ```+(src|css)/*``` is more easier.
But I'm not sure whether it is right.

Comment (PeterDaveHello) [inline ajax/libs/readremaining.js/package.json]: @dannyxx001 can you discuss with the author if it's possible to add a `dist` folder for distribution files?

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Pull request closed without merging**